Assets/Scripts/Inventory/UI/InventoryPage.cs

```diff
@@ -197,4 +208,21 @@ public void Hide()
         container.visible = false;
     }
 
+    //Return the name, rarity, and stacks of the items that the player has currently.
+    public List<Item> GetItems()
+    {
+        List<Item> playerItems = new List<Item>();
+        foreach (Item i in itemList)
+        {
+            if (i.stacks > 0)
+            {
+                //Add a new item with the name rarity and stacks of the selected item
+                playerItems.Add(new Item { name = i.name, rarity = i.rarity, stacks = i.stacks });
+            }
+        }
+        //Sort by rarity
+        playerItems.Sort((x, y) => x.rarity.CompareTo(y.rarity));
+        return playerItems;
+    }
+
```

Comment:
You should be able to change this function to pull from the new list heldItems instead of if they have stacks now